Gems/AtomLyIntegration/CommonFeatures/Code/Source/OcclusionCullingPlane/EditorOcclusionCullingPlaneComponent.cpp

```diff
@@ -32,7 +32,7 @@ namespace AZ
                     editContext->Class<EditorOcclusionCullingPlaneComponent>(
                         "Occlusion Culling Plane", "The OcclusionCullingPlane component is used to cull meshes that are inside the view frustum and behind the occlusion plane")
                         ->ClassElement(AZ::Edit::ClassElements::EditorData, "")
-                            ->Attribute(AZ::Edit::Attributes::Category, "Atom")
+                            ->Attribute(AZ::Edit::Attributes::Category, "Graphics - Lights")
```

Comment:
Shouldn't _Occlusion Culling_ be categorized into the _Mesh_ category instead of _Lights_? It is a feature that works even if there are no lights in the scene.

Reply:
Agreed, I think Mesh feels like a better fit.

Reply:
I'd have no problem with your suggestion. But it seems it is only used by Diffuse Probe Grid which is part of lighting.

Reply:
I had a conversation with @dmcdiarmid-ly , and we agreed for a new category "Graphics/Occlusion". In the future we may introduce other kind of occlusion components like "Occlusion Areas  (Volumes)" and they'd go in this category.